Sources/SnapshotTestingExtensionsDynamic

```diff
@@ -0,0 +1 @@
+/Users/oguz.yuksel/iOS_Development/Teufel/UIExtensions/Sources/SnapshotTestingExtensions
```

Comment:
this should be a relative link, not an absolute (would fail on all machines except of yours :) )

Reply:
Ahhhhhh right